qml/pages/FirstPage.qml

```diff
@@ -129,21 +129,36 @@ import Sailfish.Silica 1.0
 
           delegate: BackgroundItem {
             width: parent.width
-            height:  Theme.paddingMedium + theTitle.contentHeight
-
-            Label {
-                id:  theTitle
-                text: "<b>" + title + "</b>" + "<br>(Posts: " + posts_count + ")</br>"
-                textFormat: Text.RichText
-                wrapMode: Text.Wrap
-                font.pixelSize: Theme.fontSizeSmall
-                anchors {
-                    left: parent.left
-                    right: parent.right
-                    margins: Theme.paddingMedium
-                    verticalCenter: parent.verticalCenter
+            height:  Theme.paddingMedium + column.height
+
+            Column {
+                id: column
+                width: parent.width
+                anchors.verticalCenter: parent.verticalCenter
+
+                Label {
+                    id:  theTitle
+                    text: title
+                    wrapMode: Text.Wrap

```

Comment:
Nothing changed about wrapping. It will show whole title.

Reply:
Scalability is ok but some readability could be enhanced when the titles require more than one row since the posts label is little bit too close to the other title.

![image](http://puu.sh/GbSSn/0942b89d24.jpg)

Reply:
There seem to be some weird empty lines in your screenshot (first 3 posts with extra line that then disappears)... I'm all for removing the <b>oldening, but those look... idk weird

Reply:
![Zrzut_ekranu_20200728_001](https://user-images.githubusercontent.com/8203809/88689835-4323a680-d0fb-11ea-972d-8d02460baf9f.png) Looks good for me